homework-2/index.ts

```diff
@@ -23,10 +24,12 @@ const app: Application = express();
 const serverInstance = createServer(app);
 const { APP_PORT } = process.env;
 
+app.use(cors())
```

Comment:
would be nice to use more specific cors rules

Reply:
Done